storage/integration_test.go

```diff
@@ -7032,6 +7032,687 @@ func (te *openTelemetryTestExporter) Unregister(ctx context.Context) {
 	te.tp.Shutdown(ctx)
 }
 
+func TestIntegration_Object_Create_CustomContexts(t *testing.T) {
```

Comment:
Should stick to camelcase for the test name, e.g. TestIntegration_ObjectCreateCustomContexts